pyard/pyard.py

```
@@ -389,6 +390,9 @@ def redux_gl(self, glstring: str, redux_type: str) -> str:
         :rtype: str
         """
         
+        if not self.isvalid_gl(glstring):
+            return ""
```

Should it throw an error rather than silently failing for invalid `glstring`